templates/patterns/content_banner/pattern-content-banner.html.twig

```diff
@@ -0,0 +1,39 @@
+{#
+/**
+ * @file
+ * Content banner pattern.
+ */
+#}
+{% set _content %}
+  {% if creation_date is not empty %}
+    <div class="mb-4 mt-4">
+      <span class="text-muted me-3"> {{ creation_date }} </span>
+    </div>
+  {% endif %}
+  <p> {{ content }} </p>
+{% endset %}
+
+{% set _image = {
+  'path': image.src,
+  'alt': image.alt,
+} %}
```

Comment:
Why this extra variable here, when we can just pass this below?

Reply:
We need to / should check if the image is empty. The new version does that.
We can still use inline using ternary expression.

```twig
  'image': image ? {
    'path': image.src,
    'alt': image.alt,
  } : {},
```
I think this is more readable because we don't have to scroll around between where the variable is set and where it is used.